### PR TITLE
Spelling Updates Batch 2

### PR DESCRIPTION
**Pull request by kingthorin**
Cherry picked from: https://github.com/zaproxy/zap-extensions/pull/3083 and rebased current.

### CODE REVIEW

File: addOns/diff/src/main/java/org/zaproxy/zap/extension/diff/ZapDiffRowGenerator.java [hunk 1]
```diff
@@ -233,13 +233,13 @@ private List<String> removeBlankLines(List<String> lines) {
  public List<DiffRow> generateDiffRows(List<String> original, List<String> revised, Patch<String> patch) {
      // normalize the lines (expand tabs, escape html entities)
 	 // ZAP: Dont normalise the strings
-     //original = StringUtills.normalize(original);
-     //revised = StringUtills.normalize(revised);
+     //original = StringUtils.normalize(original);
+     //revised = StringUtils.normalize(revised);
```
Comment (thc202) [inline addOns/diff/src/main/java/org/zaproxy/zap/extension/diff/ZapDiffRowGenerator.java:237]: This should be reverted, it's not a typo the class `StringUtills` exists.

File: addOns/diff/src/main/java/org/zaproxy/zap/extension/diff/diff_match_patch.java [hunk 2]
```diff
@@ -1175,7 +1175,7 @@ public void diff_cleanupMerge(LinkedList<Diff> diffs) {
             pointer.remove();
```
Comment (thc202) [inline addOns/diff/src/main/java/org/zaproxy/zap/extension/diff/diff_match_patch.java:1175]: The changelog should be updated with the maintenances change entry.

File: addOns/domxss/src/main/java/org/zaproxy/zap/extension/domxss/DomXssScanRule.java [hunk 3]
```diff
@@ -222,7 +222,7 @@ Browser getBrowser() {
     }
```
Comment (thc202) [inline addOns/domxss/src/main/java/org/zaproxy/zap/extension/domxss/DomXssScanRule.java:222]: Same for this add-on.

File: addOns/encoder/src/main/java/org/zaproxy/addon/encoder/EncodeDecodeOptions.java [hunk 4]
```diff
@@ -67,9 +67,9 @@ public boolean isBase64DoBreakLines() {
         return base64DoBreakLines;
```
Comment (thc202) [inline addOns/encoder/src/main/java/org/zaproxy/addon/encoder/EncodeDecodeOptions.java:67]: And this one.

File: addOns/formhandler/src/main/resources/org/zaproxy/zap/extension/formhandler/resources/Messages.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
 formhandler.options.title = Form Handler
```
Comment (thc202) [inline addOns/formhandler/src/main/resources/org/zaproxy/zap/extension/formhandler/resources/Messages.properties:1]: Same.

**Comment by kingthorin** [comment]
Done x4

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**